client/matchmaking/find-1v1.tsx

```diff
@@ -98,24 +98,31 @@ const Form1v1 = React.forwardRef<FindMatchFormRef, Form1v1Props>(
             />
           </>
         ) : null}
-        <MapSelectionsHeader>
-          <SectionTitle>{t('matchmaking.findMatch.mapPool', 'Map pool')}</SectionTitle>
-          {mapPoolOutdated ? (
-            <OutdatedIndicator>{t('matchmaking.findMatch.updated', 'Updated')}</OutdatedIndicator>
-          ) : null}
-        </MapSelectionsHeader>
-        <DescriptionText>
-          {t(
-            'matchmaking.findMatch.veto1v1',
-            'Veto up to 3 maps. Vetoed maps will never be selected for play.',
-          )}
-        </DescriptionText>
-        <MapVetoesControl
-          {...bindCustom('mapSelections')}
-          mapPool={mapPool}
-          maxVetoes={3}
-          disabled={disabled}
-        />
+        {mapPool ? (
+          <>
+            <MapSelectionsHeader>
+              <SectionTitle>{t('matchmaking.findMatch.mapPool', 'Map pool')}</SectionTitle>
+              {mapPoolOutdated ? (
+                <OutdatedIndicator>
+                  {t('matchmaking.findMatch.updated', 'Updated')}
+                </OutdatedIndicator>
+              ) : null}
+            </MapSelectionsHeader>
+
+            <DescriptionText>
+              {t('matchmaking.findMatch.veto1v1', {
```

Comment:
Like I said yesterday, this message should be chosen based on whether or not the veto count allows map overlaps. If the veto count is high enough, this would be a lie, so we can't just pick this message always. Make a component that takes the veto count, map pool size, and number of players and can pick between the 2 messages.